src/mudlet.cpp

```diff
@@ -1575,32 +1575,40 @@ void mudlet::slot_timer_fires()
     if (!pQT) {
         return;
     }
-    if (mTimerMap.contains(pQT)) {
-        TTimer* pTT = mTimerMap[pQT];
+
+    // Pull the Host name and TTimer::id from the properties:
+    QString hostName(pQT->property(TTimer::scmProperty_HostName).toString());
+    if (hostName.isEmpty()) {
+        qWarning().nospace().noquote() << "mudlet::slot_timer_fires() INFO - Host name is empty - so TTimer has probably been deleted.";
```

Comment:
How would seeing this debug output help someone?

Reply:
It is a bit clearer than the "MUDLET CRITICAL ERROR ..." message from before... and will prompt me to take another look at things if it gets reported as to the circumstances under which it appeared...